project/src/main/java/com/google/sps/Models/Database.java

```diff
@@ -122,7 +122,7 @@ private static User createUser(String email, String nickname) {
     getDatastore().put(userEntity);
 
     long userID = userEntity.getKey().getId();
-    Folder defaultFolder = createFolder("default", userID);
+    Folder defaultFolder = createFolder("Root", userID);
```

Comment:
Is there any reason to set as "Root" rather than "root"?

Reply:
I named it "Root" since this name is rendered on the frontend and it looks nicer when capitalized